backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java

```diff
@@ -19,7 +19,7 @@ private static boolean isMigration() {
             Optional.ofNullable(System.getenv(CommonConstants.ENABLE_LIQUIBASE_MIGRATION_ENV_VAR))
                 .orElse("false"));
     LOGGER.info("ENABLE_LIQUIBASE_MIGRATION_ENV_VAR: {}", enableLiquibaseMigrationEnvVar);
-    return enableLiquibaseMigrationEnvVar;
+    return true;
```

Comment:
Why do we need to hardcode it? Is it a mistake?

Reply:
Yes it is by mistake committed

Reply:
reverted it.